x-pack/plugins/observability_solution/entity_manager/server/lib/entities/errors/entity_id_too_long_error.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export class EntityIdTooLong extends Error {
```

Comment:
Maybe we should be clear and call it `EntityDefinitionIdTooLong`, since it's not about a specific entity's ID

Reply:
yeh it's true, but the same is true of `EntityIdConflict` - should we rename both here?

Reply:
I would, even if it's not possible that an entity itself will fire a similar error.